network/network.go

```diff
@@ -131,22 +135,30 @@ func (c *Collector) Describe(ch chan<- *prometheus.Desc) {
 // Collect implements the Collect method of the Collector interface.
 func (c *Collector) Collect(ch chan<- prometheus.Metric) {
 	hosts := []string{}
+	atomic.AddUint64(&c.scrapeID, 1)
+
+	scrapingStart := time.Now()
+	c.logger.Log("level", "info", "message", "collecting metrics", "scrapeID", c.scrapeID)
 
 	service, err := c.kubernetesClient.CoreV1().Services(c.namespace).Get(c.service, metav1.GetOptions{})
 	if err != nil {
-		c.logger.Log("level", "error", "message", "could not get service", "stack", microerror.Stack(err))
+		c.logger.Log("level", "error", "message", "could not get service from kubernetes api ", "scrapeID", c.scrapeID, "stack", microerror.Stack(err))
```

Comment:
```suggestion
		c.logger.Log("level", "error", "message", "could not collect service", "service", c.service, "scrapeID", c.scrapeID, "stack", microerror.Stack(err))
```

Reply:
The point here is the starting point. We are getting an error from net-exporter that is timeouting when dialing a host X.X.X.X which is the kubernetes api. This is not so obvious for everybody so we decided to include some clue that this is the k8s api and maybe the problem is there (what motivated the PM)

Reply:
Ok I get your point. Then please adjust this message in 3 places and not only in the error.
`collecting service from kubernetes api`
`could not collect service from kubernetes api` 
`collected service from kubernetes api`

Reply:
done